src/components/scores/table-hs.cjsx

```diff
@@ -90,10 +90,10 @@ module.exports = React.createClass
           Review
         </Router.Link>
 
-    if heading.average
+    if heading.attempted_average
```

Comment:
Not sure if this matters, but we currently have the equivalent of total_average here (in production).